src/IO/InputStreamFromROOTFile.cxx

```diff
@@ -68,6 +69,7 @@ InputStreamFromROOTFile::set_defaults()
     singles_readout_depth = -1;
     exclude_scattered = false;
     exclude_randoms = false;
+    check_energy_window_information = true;
```

Comment:
do we document default values of all these `bool`s somewhere? Should be in the doxygen, I guess best for `set_defaults()` (can put it in the .cxx).

Also, the class doc in the .h files needs to have this new variable (and refer to `set_defaults()` for more info. Finally, the `examples/samples/*.hroot` files need to have it.

(you can see why I thought this was some work... 😄 )